Add custom service Full AC state for Sensibo (#24666)
home-assistant/home-assistant.github.io: source/_integrations/sensibo.markdown

## Custom Services

### Full state

> You can send a full state command to Sensibo instead of single commands by using the service `sensibo.full_state`.

Reviewer: ```suggestion
You can send a full state command to Sensibo instead of single commands using the service `sensibo.full_state`.
```

> All fields are required to be according to Sensibo api specification and is case sensitive.

Reviewer 2: ```suggestion
All fields are required to be according to Sensibo API specifications and are case-sensitive.
```

> To see the options for each field to use by this service:

Reviewer: ```suggestion
To see the options for each field to use this service:
```

1. Switch to the relevant HVAC mode (not all HVAC modes have the same options)

> 2. Retrieve the options for `fan_modes` and `swing_modes` from the climate entity's attributes

Reviewer 2: ```suggestion
1. Switch to the relevant HVAC mode (not all HVAC modes have the same options).
2. Retrieve the options for `fan_modes` and `swing_modes` from the climate entity's attributes.
```
